pkg/objectstate/rte/rte.go

```diff
@@ -257,11 +260,64 @@ func (em *ExistingManifests) State(mf rtemanifests.Manifests, updater GenerateDe
 	}
 
 	for _, tree := range em.trees {
-		for _, mcp := range tree.MachineConfigPools {
+		if em.plat == platform.OpenShift {
```

Comment:
we can merge to have actually CI running, but this code is now untenable, we need to plan a cleanup/possibly a redesign ASAP